utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -54,8 +54,8 @@ namespace fs = boost::filesystem;
 #define DEBUG_MODE 0
 #define MAX_IMAGE_DUMP 20
 #define MAX_BATCH_SIZE 512
-#define GOLDEN_OUTPUT_MAX_HEIGHT 150    // Golden outputs are generated with MAX_HEIGHT set to 150. Changing this constant will result in QA test failures
-#define GOLDEN_OUTPUT_MAX_WIDTH 150     // Golden outputs are generated with MAX_WIDTH set to 150. Changing this constant will result in QA test failures
+#define GOLDEN_OUTPUT_MAX_HEIGHT 224    // Golden outputs are generated with MAX_HEIGHT set to 150. Changing this constant will result in QA test failures
+#define GOLDEN_OUTPUT_MAX_WIDTH 224     // Golden outputs are generated with MAX_WIDTH set to 150. Changing this constant will result in QA test failures
```

Comment:
The comment says its set to 150, but is changed to 224 in the code. This will cause failure for other funcs.
Why is this 224?

Reply:
Else we might need to add 3 images set with 150x150 as dimensions to test RICAP alone.
Please let me know on your comments.

Reply:
@snehaa8 Okay so does the test suite currently work with these changes?